Sources/DashX/DashXClient.swift

```diff
@@ -97,11 +93,12 @@ public class DashXClient {
     // MARK: -- identify
 
     public func identify(_ uid: String?, withOptions: NSDictionary?) throws {
+        /* MARK: - Why do we return here
         if uid != nil {
             self.uid = uid
             return
         }
-
+         */
```

Comment:
- Add setIdentity
- Remove `uid` as a parameter
- Pick `uid` from options["uid"] if present, or else use self.uid
- Same goes for `anonymousUid` - pick from options["anonymousUid"] if present, or else use self.anonymousUid
-